test/uk/gov/hmrc/cdsreimbursementclaim/models/tpi01/ClaimsResponseSpec.scala

```diff
@@ -0,0 +1,22 @@
+package uk.gov.hmrc.cdsreimbursementclaim.models.tpi01
+
+import org.scalatest.matchers.should.Matchers
+import org.scalatest.wordspec.AnyWordSpec
+
+class ClaimsResponseSpec extends AnyWordSpec with Matchers {
+
+  case class SampleClass(id: Int, name: String)
+
+  "ClaimResponse" should {
+    "removeDuplicates" in {
+      val list = Seq(SampleClass(1, "Bob"), SampleClass(1, "Bob"), SampleClass(2, "Charlie"))
```

Comment:
I would prefer to preserve original ordering here

Reply:
Addressed the comment